src/smexperiments/api_types.py

```diff
@@ -272,6 +272,85 @@ def __init__(self, code=None, message=None, metric_index=None, **kwargs):
         super(BatchPutMetricsError, self).__init__(code=code, message=message, metric_index=metric_index, **kwargs)
 
 
+class TrainingJobSearchResult(_base_types.ApiObject):
+    """Summary model of an Training Job search result.
+
+    Attributes:
+        training_job_name (str): The name of the training job.
+        training_job_arn (str): The Amazon Resource Name (ARN) of the training job.
+        tuning_job_arn (str): The Amazon Resource Name (ARN) of the associated.
+        hyperparameter tuning job if the training job was launched by a hyperparameter tuning job.
+        labeling_job_arn (str): The Amazon Resource Name (ARN) of the labeling job.
+        autoML_job_arn (str): The Amazon Resource Name (ARN) of the job.
+        model_artifacts (dict): Information about the Amazon S3 location that is configured for storing model artifacts.
+        training_job_status (str): The status of the training job
+        hyper_parameters (dict): Algorithm-specific parameters.
+        algorithm_specification (dict): Information about the algorithm used for training, and algorithm metadata.
+        input_data_config (dict): An array of Channel objects that describes each data input channel.
+        output_data_config (dict): The S3 path where model artifacts that you configured when creating the job are
+        stored. Amazon SageMaker creates subfolders for model artifacts.
+        resource_config (dict): Resources, including ML compute instances and ML storage volumes, that are configured
+        for model training.
+        debug_hook_config (dict): Configuration information for the debug hook parameters, collection configuration,
+        and storage paths.
+        debug_rule_config (dict): Information about the debug rule configuration.
+    """
+
+    training_job_name = None
+    training_job_arn = None
+    tuning_job_arn = None
+    labeling_job_arn = None
+    autoML_job_arn = None
+    model_artifacts = None
+    training_job_status = None
+    hyper_parameters = None
+    algorithm_specification = None
+    input_data_config = None
+    output_data_config = None
+    resource_config = None
+    debug_hook_config = None
+    experiment_config = None
+    debug_rule_config = None
```

Comment:
how did you construct this list of fields? what about some other fields like  SecondaryStatus and FailureReason?

Reply:
I added the ones I think are most useful, but others should be covered by **kwargs

Reply:
seems incomplete tho, we will get back all the fields in search regardless of what is in kwargs. i guess we can follow up later and add the remaining fields.